docs/ref/files/file.txt

```diff
@@ -56,6 +56,9 @@ The ``File`` class
         was originally opened with; ``None`` means to reopen with the original
         mode.
 
+        Returns ``self``, so that it can be used similar to Python's
```

Comment:
This should have a `.. versionchanged:: 2.0` annotation as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.

This could be considered a "minor feature" and mentioned in `docs/releases/2.0.txt` release notes but I wouldn't require it.